scripts/init.js

```diff
@@ -43,12 +45,31 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
     JSON.stringify(appPackage, null, 2)
   );
 
+  var readmeExists = pathExists.sync(path.join(appPath, 'README.md'));
+  if (readmeExists) {
+    fs.renameSync(path.join(appPath, 'README.md'), path.join(appPath, 'README.old.md'));
+  }
+
   // Copy the files for the user
   fs.copySync(path.join(ownPath, 'template'), appPath);
 
   // Rename gitignore after the fact to prevent npm from renaming it to .npmignore
   // See: https://github.com/npm/npm/issues/1862
-  fs.move(path.join(appPath, 'gitignore'), path.join(appPath, '.gitignore'), []);
+  fs.move(path.join(appPath, 'gitignore'), path.join(appPath, '.gitignore'), [], function (err) {
+    if (err) {
+      // Append if there's already a `.gitignore` file there
+      if (err.code === 'EEXIST') {
+        fs.readFile(path.join(appPath, 'gitignore'), (err, data) => {
+          if (err) throw err;
+          fs.appendFileSync(path.join(appPath, '.gitignore'), data);
+          fs.unlinkSync(path.join(appPath, 'gitignore'));
+        });
+      }
```

Comment:
Sigh! sorry for these silly mistakes. I thought `eslint` had those covered. Didn't bother checking coding style since there's a `.eslintrc.js` in the root.